electrum/three_keys/utils.py

```diff
@@ -0,0 +1,41 @@
+from typing import Tuple
+
+from .tx_type import TxType
+
+
+TX_STATUS_INDEX_SHIFT = 10
+TX_TYPES_LIKE_STANDARD = (
+    TxType.NONVAULT,
+    TxType.INSTANT,
+    TxType.ALERT_CONFIRMED,
```

Comment:
why `ALERT_PENDING` and `ALERT_RECOVERED` are missing?

Reply:
this tuple contains types which will be displayed like NonValut transaction, so `ALERT_PENDING` and `ALERT_RECOVERED` should be missed